cloud/deploys/staging/main.tf

```diff
@@ -21,4 +25,10 @@ module "app" {
 
   application_name = var.application_name
   app_secret_key   = var.app_secret_key
+  ses_sender_email = var.sender_email_address
+}
+
+module "email_service" {
+  source               = "../../aws/modules/ses"
+  sender_email_address = var.sender_email_address
```

Comment:
suggestion (non-blocking, if-minor): could we make using SES with Azure optional on a config value? very much an only-if-minor suggestion

Reply:
i would think that this file would change if we were going to the deploy differently if that makes sense not passing in the config values here but fully just not having this exists in the deploys folder